src/Tribe/Template.php

```diff
@@ -687,6 +725,24 @@ public function template( $name, $context = array(), $echo = true ) {
 		 */
 		$html = apply_filters( 'tribe_template_html', $html, $file, $name, $this );
 
+		/**
+		 * Allow users to filter the final HTML by the name
+		 *
+		 * E.g.:
+		 *    `tribe_template_html:events/blocks/parts/details`
+		 *    `tribe_template_html:events/embed`
+		 *    `tribe_template_html:tickets/login-to-purchase`
+		 *
+		 * @deprecated   4.11.0
+		 * @since  4.7.20
+		 *
+		 * @param string $html      The final HTML
+		 * @param string $file      Complete path to include the PHP File
+		 * @param array  $name      Template name
+		 * @param self   $template  Current instance of the Tribe__Template
+		 */
+		$html = apply_filters( "tribe_template_html:$legacy_hook_name", $html, $file, $name, $this );
```

Comment:
:warning: Words in hook names should be separated using underscores. Expected: "tribe_template_html_$legacy_hook_name", but found: "tribe_template_html:$legacy_hook_name".

<sub>hash: e8765237436082873bd0ae952b8d2345</sub>